auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java

```diff
@@ -270,6 +271,9 @@ public boolean isAzureDiskSSEWithCMKEnabled(String accountId) {
         return isEntitlementRegistered(accountId, CDP_CB_AZURE_DISK_SSE_WITH_CMK);
     }
 
+    public boolean isAWSDiskEncryptionWithCMKEnabled(String accountId) {
+        return  isEntitlementRegistered(accountId, CDP_CB_AWS_DISK_ENCRYPTION_WITH_CMK); }
```

Comment:
Please fix the spacing (2 spaces after `return`) and move closing brace on new line.

Reply:
LGTM.